stubs/app/server/lib/logins.ts

```diff
@@ -1,6 +0,0 @@
-import { getCoreLoginSystem } from "app/server/lib/coreLogins";
```

Comment:
(We will need to be careful when syncing SaaS, also grist-static, since they are still using logins.ts)